### DISCUSSION TITLE
Metadynamics - what is "kpush" exactly?

### DISCUSSION THREAD

**Discussion by johnwkeller54** [grimme-lab/xtb#1094] [General]
If two atoms are listed, does a "kpush=0.02" statement push atoms together, or pull them apart? Also, the examples show kpush values of either 0.02 or 1.0. What is the effect of different values (e.g., larger = stronger push?)

**Comment by mattgummow** [comment]
Hi @johnwkeller54, did you ever find out what kpush is? I am struggling with a similar problem when running metadynamics. The xcontrol.7.adoc lists kpush as 'scaling factor for rmsd criteria can be positive and negative'.
